cmd/tapcli/assets.go

```diff
@@ -310,29 +333,82 @@ var finalizeBatchCommand = cli.Command{
 				"the minting transaction",
 		},
 	},
-	Action: finalizeBatch,
+	Action: fundBatch,
 }
 
-func parseFeeRate(ctx *cli.Context) (uint32, error) {
-	if ctx.IsSet(feeRateName) {
-		userFeeRate := ctx.Uint64(feeRateName)
-		if userFeeRate > math.MaxUint32 {
-			return 0, fmt.Errorf("fee rate exceeds 2^32")
-		}
+func fundBatch(ctx *cli.Context) error {
+	ctxc := getContext()
+	client, cleanUp := getMintClient(ctx)
+	defer cleanUp()
 
-		// Convert from sat/vB to sat/kw. Round up to the fee floor if
-		// the specified feerate is too low.
-		feeRate := chainfee.SatPerKVByte(userFeeRate * 1000).
-			FeePerKWeight()
+	feeRate, err := parseFeeRate(ctx)
+	if err != nil {
+		return err
+	}
 
-		if feeRate < chainfee.FeePerKwFloor {
-			feeRate = chainfee.FeePerKwFloor
-		}
+	resp, err := client.FundBatch(ctxc, &mintrpc.FundBatchRequest{
+		ShortResponse: ctx.Bool(shortResponseName),
+		FeeRate:       feeRate,
+	})
+	if err != nil {
+		return fmt.Errorf("unable to fund batch: %w", err)
+	}
 
-		return uint32(feeRate), nil
+	printRespJSON(resp)
+	return nil
+}
+
+var sealBatchCommand = cli.Command{
+	Name:        "seal",
+	Usage:       "seal a batch",
+	Description: "Attempt to seal the pending batch.",
```

Comment:
Does this command really do anything on the command line other than sanity check that we can generate witnesses for all group keys in the batch? Not sure if we should hide (`Hidden: true`) the command and only keep it for debugging? Since it probably doesn't make sense to support submitting the actual witnesses through the CLI (only makes sense through RPC).

Reply:
Yeah I agree it doesn't have much use...

I suppose we want to still expose `fund` as we'll add other features to that later.

Reply:
Update: added more to the descriptions and hid this cmd. In practice you could do `fund` + `finalize` from the CLI if we don't accept witness in `seal`.